apps/api/src/app/question-management/dtos/answer.dto.ts

```diff
@@ -0,0 +1,15 @@
+export class AnswerDto {
+  id?: string;
+  content: string;
```

Comment:
**suggestion:** As reminder, please annotate all your DTO's with the Swagger annotation (description and example)